content/en/docs/installation/authentication.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Authentication"
+weight: 40
+description: >
+  Guide to set up basic authentication with Fission
+---
+
+## Authentication in Fission
+
+When using Fission, if you are using an ingress, you might already have some form of authentication in place for external calls.
+But if you aren't, Fission didn't provide a way to configure authentication for your API calls.
+
+Starting with v1.16.0, Fission allows you to have an authentication mechanism in place.
```

Comment:
We can link RC release notes to version as for now.

Reply:
Linked